openstack/cerebro/cerebro-main/templates/dc-state-service.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dc-state-service
+  labels:
+    app: dc-state-service
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "{{ .Values.dcStateService.prometheusClientPort }}"

```

Comment:
which prometheus is scraping this? If it is the same as for vrops, we've got to define it in the region secrets.

`prometheus.io/targets: {{ required ".Values.alerts.prometheus missing" .Values.alerts.prometheus | quote }}`

add a default for like

```yaml
alerts:
  enabled: false
  prometheus: some-random-prometheus-url
```

Don't bother the alerts yet. If we have alerts at some point, we've got to enable them in the regions and need to do respective routing beforehand. It doesn't hurt, to have it in here tho.